pkg/kn/commands/plugin/handler.go

```diff
@@ -76,6 +76,16 @@ func (h *DefaultPluginHandler) Lookup(name string) (string, bool) {
 			return pluginDirPluginPath, true
 		}
 
+		// Try to match well-known file extensions on Windows
+		if runtime.GOOS == "windows" {
+			for _, ext := range []string{".bat", ".cmd", ".com", ".exe", ".ps1"} {
+				pathWithExt := pluginDirPluginPath + ext
+				if _, err = os.Stat(pathWithExt); !os.IsNotExist(err) {
```

Comment:
What if `err != nil` but also `!os.IsNotExist()` is true ? (i.e. if there is an error as than not-found). The path is then returned nevertheless, which imo is wrong. (same error btw in the check above, too).

IMO this method should return `(string,err)` with an empy string if nothing is found, non-empty string if the plugin is found and err if any other error occured.

Reply:
Also just split the `os.Stat` and `os.IsNotExit` to make logic clear

Reply:
For now I've opted for less invasive solution that will only check `err == nil` to fix the reported issue. And also allow straightforward cherry-pick to  `release-0.13`. 

I agree that we should look closer to handler code and improve Lookup as suggested by @rhuss and anything else that will come up.